src/plugin-host/plugin/plugin-command.ts

```diff
@@ -0,0 +1,21 @@
+
+/**
+ * Defines an interface for plugins
+ * to register their own custom commands
+ * with appropriate callbacks.
+ * 
+ * A Plugin Command must have some additional output.
+ */
+export class PluginCommand
```

Comment:
Can't this use or rely on the existing DankTimesBotCommand class?

Reply:
Yes